src/openrct2/ride/coaster/meta/LayDownRollerCoaster.h

```diff
@@ -60,7 +60,7 @@ constexpr const RideTypeDescriptor LayDownRollerCoasterAltRTD =
     SET_FIELD(AlternateType, RIDE_TYPE_NULL),
     SET_FIELD(Category, RIDE_CATEGORY_NONE),
     SET_FIELD(EnabledTrackPieces, {TRACK_STRAIGHT, TRACK_FLAT_ROLL_BANKING, TRACK_SLOPE, TRACK_SLOPE_STEEP_UP, TRACK_SLOPE_STEEP_DOWN, TRACK_SLOPE_CURVE, TRACK_SLOPE_CURVE_STEEP, TRACK_S_BEND, TRACK_CURVE_SMALL, TRACK_CURVE, TRACK_HELIX_LARGE, TRACK_BRAKES, TRACK_ON_RIDE_PHOTO, TRACK_BLOCK_BRAKES, TRACK_INLINE_TWIST_INVERTED, TRACK_FLYING_HALF_LOOP_INVERTED_DOWN, TRACK_CORKSCREW_INVERTED}),
-    SET_FIELD(ExtraTrackPieces, {TRACK_FLYING_HALF_LOOP_INVERTED_UP}),
+    SET_FIELD(ExtraTrackPieces, {TRACK_FLYING_HALF_LOOP_INVERTED_UP, TRACK_STATION_END}),
```

Comment:
Will you be adding the large steep slope pieces from the Corkscrew/Hypercoaster to the Lay-down too?

Reply:
I forgot those existed. Probably better not to include them in this PR.

Reply:
This is really easy to add with a single line. It's the same line I used to make the Looping Coaster draw the missing elements from the LIM Launch coaster. 
![image](https://user-images.githubusercontent.com/1516893/184518530-87f24156-b938-4748-aacd-583908adbed8.png)

Reply:
The booster track piece is the last remaining track piece from the Corkscrew/Hypercoaster that isn't present on the Lay-down RC currently.